lib/network/http2_network.go

```diff
@@ -223,16 +135,21 @@ func (t *HTTP2Network) setNotReadyHandler() {
 	t.server.Handler = handlers.CombinedLoggingHandler(t.config.HTTP2LogOutput, handler)
 }
 
-func (t *HTTP2Network) AddHandler(ctx context.Context, pattern string, handler func(context.Context, *HTTP2Network) HandlerFunc) (err error) {
-	t.handlers[pattern] = handler(ctx, t)
+type Re interface {
+	ResponseMessage(http.ResponseWriter, string)
+	ReceiveMessage(*HTTP2Network, Message)
+}
+
```

Comment:
IMHO, The name `Re` is too short [for](url) what it is. :-) `Receiver`?

Reply:
@anarcher Oh, sorry.. `Re` is prefix of `ResponseMessage` and `ReceiveMessage`. I think `Receiver` is good also :)

Reply:
Maybe it could be my personal taste.... but I feel the **interface** has two feature...  I would like to design the interface like below: 

```
type MessageReceiver interface {
        Receive(Message)
}

type ResponseWriter struct {
       Write(string) // ? 
} 
```

And IMHO, I think that the intefaces' implemantation can be variables of the `HTTP2Network`. Doesn't this make sense? (but it is my personal taste.. :-D)

Reply:
@anarcher  I don't understand well. Could you explain more?

Reply:
@astonshnam Sometimes, I am not entirely sure, but  **interface** ensure implementations honor a contract. If it is a separate usage case, just I feel two use cases (receive messages to consense, return HTTP response). However, it is not essential. Almost I don't make one use-case interface for all situation. 
IMHHHO, For interface consumer's view, Don't need to know except Message, HTTP response.

It is a just personal opinion. :-D

Reply:
@anarcher @Charleslee522 `Re` and `HttpRe` is also not my taste :) My suggestion is `MessageBroker` or `MessageGate`?

Reply:
My choosing term was `MessageReceiver`,  but `MessageBroker` is better than my term! :-)

Reply:
@spikeekips @anarcher I agree with `MessageBroker`, then variable name of MessageBroker `mb`, write?

Reply:
@Charleslee522 good choice.

Reply:
@anarcher Please let me know what you wrote, anytime you want :)